### DISCUSSION TITLE
Restrict One Repository in Organization

### DISCUSSION THREAD

**Discussion by jdh313** [community/community#30132] [New to GitHub] [answered]
Hi,

We've recently moved our organization to GitHub. Our base permission is for write access, as we're a small team. We have one repository, however, that we only want certain people to be able to view, much less write. Is this possible?

Thanks

**Comment by GabyCT** [comment, marked as answer]
Yes, you can find all the info here https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository
